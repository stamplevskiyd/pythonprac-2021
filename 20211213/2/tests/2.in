[15, 83, 86, 13, 86, 2, 88, 38, 63, 50, 53, 80, 52, 92, 29, 94, 62, 15, 25, 67, 8, 92, 67, 54, 74, 17, 41, 27, 8, 23, 93, 37, 10, 12, 5, 13, 79, 54, 51, 5, 56, 13, 15, 3, 68, 75, 8, 94, 58, 65, 36, 31, 29, 52, 49, 52, 11, 76, 13, 30, 43, 75, 41, 45, 81]