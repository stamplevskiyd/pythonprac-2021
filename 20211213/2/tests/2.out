[2, 3, 5, 5, 8, 8, 8, 10, 11, 12, 13, 13, 13, 13, 15, 15, 15, 17, 23, 25, 27, 29, 29, 30, 31, 36, 37, 38, 41, 41, 43, 45, 49, 50, 51, 52, 52, 52, 53, 54, 54, 56, 58, 62, 63, 65, 67, 67, 68, 74, 75, 75, 76, 79, 80, 81, 83, 86, 86, 88, 92, 92, 93, 94, 94]
